tests/data/mean-pk/A-pkAnalysis.csv

```diff
@@ -1,3 +1,3 @@
 "Path","Parameter","Value","Unit"
-"Organism|A|Concentration in container","C_max",2.85019207000732,"µmol/l"
-"Organism|A|Concentration in container","AUC_tEnd",3292.13500976563,"µmol*min/l"
+"Organism|A|Concentration in container","C_max",2.85,"µmol/l"

```

Comment:
Hum... Why are those values truncated? Number  of digits is only a display artefact not an `export` artefact. This looks suspicious to me